### PR TITLE
IT-3924: Setup sso admin group

### PR DESCRIPTION
**Pull request by xschildw**
Sets up the sso admin group for Synapse LLM dev account.

Depends on #1227 .

### CODE REVIEW

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -2070,3 +2075,20 @@ SsoOpenchallengesProdAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref OpenchallengesProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoSynapseLlmDevAdmin:
+  Type: update-stacks
+  DependsOn: SsoAdministrator
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:2082]: i see that the latest one in this file is `v0.7.7`, i would use that one.

File: org-formation/700-aws-sso/_tasks.yaml [hunk 2]
```diff
@@ -2070,3 +2075,20 @@ SsoOpenchallengesProdAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref OpenchallengesProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoSynapseLlmDevAdmin:
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:2079]: this is the `personal AWS accounts` section.  please move this about personal accounts.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (xschildw) [reply org-formation/700-aws-sso/_tasks.yaml:2082]: OK

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (xschildw) [reply org-formation/700-aws-sso/_tasks.yaml]: Moved with the other Synapse* access

**Review by zaro0508 — APPROVED** [review]

**Comment by xschildw** [comment]
Closing this PR and replacing by same on prod account.

**Pull request closed without merging**